recipes/mirror_maker.rb

```diff
@@ -0,0 +1,93 @@
+# coding: UTF-8
+# Cookbook Name:: cerner_kafka
+# Recipe:: mirror_maker
+
+# Create an array for storing errors we should verify before doing anything
+errors = Array.new
+
+# Verify node["kafka"]["mirror_maker"] is setup properly
+ruby_block 'assert mirror maker config is correct' do # ~FC014
+  block_name 'attribute_mirrormaker_assertions'
+  block do
+    # Sources
+    if node['kafka']['mirror_maker']['mirror_sources'].to_a.empty?
+      errors.push 'node[:kafka][:mirror_maker][:mirror_sources]  must not be empty'
+    else
+      srcs = node['kafka']['mirror_maker']['mirror_sources']
+      srcs.each do |c|
+        warn c
```

Comment:
I think so. I tried to follow the style of the main kafka recipe (since it more or less is, w/ different java class), as seen here: https://github.com/cerner/cerner_kafka/blob/v1.1.0/recipes/default.rb#L5-L28

I'm not particularly attached to it, but the assertions match the expected behaviour for a mirror maker configuration. I see I need to update the error message emitted to match the recipe name, and will include that.